src/action.py

```diff
@@ -0,0 +1,16 @@
+from .object import Object
```

Comment:
Do you think we could give `./usr` directory more structure? It is becoming difficult to read through.

Reply:
We should